model/riscv_sys_regs.sail

```diff
@@ -108,35 +108,36 @@ function clause currentlyEnabled(Ext_Sstc) = hartSupports(Ext_Sstc)
 val ext_veto_disable_C : unit -> bool
 
 function legalize_misa(m : Misa, v : xlenbits) -> Misa = {
-  let  v = Mk_Misa(v);
+  let v = Mk_Misa(v);
   /* Suppress updates to MISA if MISA is not writable or if by disabling C next PC would become misaligned or an extension vetoes */
-  if   not(sys_enable_writable_misa()) | (v[C] == 0b0 & (nextPC[1] == bitone | ext_veto_disable_C()))
-  then m
-  else {
-    /* Suppress enabling C if C was disabled at boot (i.e. not supported) */
-    let m = if not(hartSupports(Ext_C)) then m else [m with C = v[C]];
-    /* Suppress updates to misa.{f,d} if disabled at boot */
-    if   not(hartSupports(Ext_F))
-    then m
-    else [m with F = v[F], D = v[D] & v[F]]
-  }
+  if   not(sys_enable_writable_misa()) | (v[C] == 0b0 & (nextPC[1] == bitone | ext_veto_disable_C())) then m
```

Comment:
```suggestion
  if   not(sys_enable_writable_misa()) | (v[C] == 0b0 & (nextPC[1] == bitone | ext_veto_disable_C()))
  then m
```
I find it easier to read when the `then` is broken out from a long predicate like this one